desktop/src/main/java/org/destinationsol/desktop/SolDesktop.java

```diff
@@ -11,55 +26,96 @@
 import org.slf4j.LoggerFactory;
 import org.terasology.crashreporter.CrashReporter;
 
-import java.io.*;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
 import java.nio.charset.Charset;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.text.SimpleDateFormat;
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.Collections;
 import java.util.Date;
 import java.util.List;
+import java.util.stream.Stream;
+
+/**
+ * This class is the desktop entry point for the whole DestinationSol application. It handles the creation and launching
+ * of LwjglApplication from {@link SolApplication}.
+ */
+public final class SolDesktop {
 
-public class SolDesktop {
     private static Logger logger = LoggerFactory.getLogger(SolDesktop.class);
 
+    /**
+     * Specifies the commandline option to pass to the application for it to generate no crash reports. Coincidentally,
+     * it is the only commandline option detected by the application.
```

Comment:
Saying it is the only command line option isn't really very useful, and is just text that would end up getting removed if we ever add another :-) Unless somebody then forgets to remove it!